src/pkg/inspectors/riskmanager/config.go

```diff
@@ -20,17 +19,15 @@ func getEnv(key, defaultValue string) string {
 // DefaultConfig read configuration from env
 func DefaultConfig() *Config {
 	return &Config{
-		Server:      "localhost:8080",
-		DetailIndex: "risk_manager_details",
-		StandAlone:  true,
+		Server:     "localhost:8080",
+		StandAlone: true,
 	}
 }
 
 // ReadEnvConfig read configuration from env
 func ReadEnvConfig() *Config {
 	return &Config{
-		Server:      getEnv("SERVER_ADDR", "localhost:8080"),
-		DetailIndex: getEnv("DETAIL_INDEX", "risk_manager_details"),
-		StandAlone:  getEnv("STANDALONE", "false") == "true",
+		Server:     getEnv("SERVER_ADDR", "localhost:8080"),
+		StandAlone: getEnv("STANDALONE", "false") == "true",
```

Comment:
Scanner should have no aware of the details of the report in opensearch.

Scanner should only care about:
1. How to generate the report.
2. How to send to exporter.

Exporter should care about:
1. How to forward the report to the consumers.

Here "index" is a nothing in consumer, of course scanner should have no control on that.

After the change exporter will add the index automatically based on the `Source` field. For example:
1. When Source is "image-scanner", then exporter will set the index of OpenSearch to "image-scanner" automatically.